examples/text-sentiment/text_sentiment/runtime_model/hf_module.py

```diff
@@ -16,7 +16,7 @@
 import os
 
 # Third Party
-from transformers import pipeline
+from transformers import pipeline  # pylint: disable=import-error
```

Comment:
I'm curious why we need to import this. Is there an import-time side effect that we need? If so, it would be good to have a comment to that effect

Reply:
It is imported because pipeline is used directly in the example init() [L51](https://github.com/caikit/caikit/pull/204/files#diff-3cc4fc99fd1cd93a8820c146bdcd7997619a6346e3a8df00df3d12f0b90b0a8cR51),  but since examples its own requirements.txt and we lint at the project level this is disabled.

Reply:
aaaah, sorry, I mixed this up with an unused import. Carry on!